paymentsheet/src/main/java/com/stripe/android/paymentsheet/elements/common/TextFieldElement.kt

```diff
@@ -14,14 +14,15 @@ import kotlinx.coroutines.flow.map
  */
 internal class TextFieldElement(
     private val textFieldConfig: TextFieldConfig
-) {
+) : Element {
     @StringRes
-    val label: Int = textFieldConfig.label
+    override val label: Int = textFieldConfig.label
     val debugLabel = textFieldConfig.debugLabel
 
     /** This is all the information that can be observed on the element */
     private val _input = MutableStateFlow("")
     val input: Flow<String> = _input
+    override val paymentMethodParams = _input
```

Comment:
What's the difference between `input` and `paymentMethodParams`?

Reply:
Removed references to paymentMethodParams in all classes depended on by Form